people.rkt

```diff
@@ -483,8 +483,8 @@ software.
                       "Joined Northeastern, 2023"
                       "B.S. University of Maryland"]
                   #:website "https://tybug.github.io/"
-                  #:img "Liam_DeVoe.jpg"]{
-            I'm interested in property based testing, open-source ecosystems, formal verification, and testing of all kinds.
+                  #:img "liam_devoe.jpg"]{
```

Comment:
@tybug this change of the case in the filename breaks the link to your photo on the People page.

Reply:
oops, thanks! I did actually change this locally but apparently [git is weird about case-sensitive changes](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git).